contrib/gitian-build.py

```diff
@@ -26,12 +27,18 @@ def setup():
         subprocess.check_call(['git', 'clone', 'https://github.com/bitcoin-core/gitian.sigs.git'])
     if not os.path.isdir('bitcoin-detached-sigs'):
         subprocess.check_call(['git', 'clone', 'https://github.com/bitcoin-core/bitcoin-detached-sigs.git'])
-    if not os.path.isdir('gitian-builder'):
+    if not os.path.isdir('gitian-builder/.git'):
+        if os.path.isdir('gitian-builder/inputs'):
+            shutil.move('gitian-builder/inputs', 'gitian-builder-inputs')
         subprocess.check_call(['git', 'clone', 'https://github.com/devrandom/gitian-builder.git'])
+        if os.path.isdir('gitian-builder-inputs'):
+            shutil.move('gitian-builder-inputs', 'gitian-builder/inputs')
     if not os.path.isdir('bitcoin'):
         subprocess.check_call(['git', 'clone', 'https://github.com/bitcoin/bitcoin.git'])
     os.chdir('gitian-builder')
-    make_image_prog = ['bin/make-base-vm', '--suite', 'bionic', '--arch', 'amd64']
+    distro = subprocess.check_output(['lsb_release', '-is']).decode("utf-8").strip('\n').lower()
+    suite = subprocess.check_output(['lsb_release', '-cs']).decode("utf-8").strip('\n')
+    make_image_prog = ['bin/make-base-vm', '--distro', distro, '--suite', suite, '--arch', 'amd64']
```

Comment:
Do not move from `bionic`. Ref: #12511

Reply:
Ah... ok, then I will PR to correct the documentation to use ubuntu/bionic.
Thanks @hebasto !
And debian/stretch?

Reply:
> The documentation suggest to use debian 8.x, but the script bin/make-base-vm is invoked with '--suite bionic’ that on debian rises an error 

You should upgrade your lxc >= 2.1.1 and debootstrap >= 1.0.92

Reply:
Do not change this line, revert your changes. The VM that is built is for the gitian build and it's distro and suite have nothing to do with the host OS. The same distro and suite must be used in order to have the same binaries be built.

Reply:
gitianuser@debian:~$ dpkg-query -W lxc debootstrap
debootstrap	1.0.89
lxc	1:2.0.7-2+deb9u2
gitianuser@debian:~$ lsb_release -a
No LSB modules are available.
Distributor ID:	Debian
Description:	Debian GNU/Linux 9.5 (stretch)
Release:	9.5
Codename:	stretch

@ken2812221  It seem that debian 9.x does not provide it...

Reply:
@cisba You can use ubuntu bionic or self-compile and install them from source.

Reply:
See the note in section https://github.com/bitcoin-core/docs/blob/master/gitian-building.md#preparing-the-gitian-builder-host

Reply:
@MarcoFalke I followed it but does not work because bionic is hard coded in gitian scripts.
I tried changing it but others errors rise so I quit for a wile debian vm.

Reply:
@hebasto comment is the more remarkable IMHO